### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by monkeygroover**
Fixes unused variable and deprecation warnings.

Note there are a lot of deprecations from old versions of dependencies, to fix these requires elixir 1.9+

### CODE REVIEW

**Comment by monkeygroover** [comment]
Looks like the CI infra will need bumping too for this to work?

**Comment by Adriandmen** [comment]
Indeed. I just chose the same versions that were used in the old ,travis.yml file. Bumping these versions up should do the trick.

File: mix.exs [hunk 1]
```diff
@@ -5,12 +5,12 @@ defmodule Osabie.MixProject do
     [
       app: :osabie,
       version: "1.0.1",
-      elixir: ">= 1.6.0",
+      elixir: ">= 1.9.0",
```
Comment (Adriandmen) [inline mix.exs:8]: The version in the [workflow.yml](https://github.com/Adriandmen/05AB1E/blob/master/.github/workflows/workflow.yml#L23) needs to be adjusted for the CI to work properly.

The minimum supported version in the **README.md** should also be updated as well.

**Review by Adriandmen — CHANGES_REQUESTED** [review]
Everything looks good to me! Just minor comments regarding the Elixir version.

**Review by Adriandmen — APPROVED** [review]

**Comment by Adriandmen** [comment]
Looks good to me, thanks! Dismissing the **codecov** checks, those are irrelevant.

**Comment by monkeygroover** [comment]
not sure where is best to discuss 05ab1e stuff, so i'll just leave this here.

I'm playing around with an elixir livebooks for interactive 05ab1e experimentation, if i get anything useful working i'll post it somewhere.

also thanks for creating 05ab1e!

**Pull request merged**